backup

```diff
@@ -30,7 +31,8 @@ case "$1" in
 			-x \
 			${OPT_BORG} ${OPT_BORG_CREATE} \
 			${EXCLUDE_FILE:+--exclude-from=${EXCLUDE_FILE}} \
-			::'{hostname}'${SUFFIX}'-{now:%Y-%m-%d_%H:%M}' \
+			--exclude-if-present .nobackup \
```

Comment:
What about putting this into `example.exclude`? I'm a big fan of not hardcoding such things.

Reply:
That's fair; I'll add a variable that pulls from example.exclude.

Reply:
So, this will have to be a separate file as it cannot be contained within the EXCLUDE_FILE variable. I'll make the change and update the pull request.

-exclude-if-present documentation says "exclude directories that are tagged by containing a filesystem object with the given NAME". For what I included in the script, if '.nobackup' is present in a directory, it will exclude all files in that directory and subdirectories.

Reply:
Disregard; I'll make this a variable in example.env. I can't imagine this having more than a couple of filenames for this type of tag, and a variable makes sense rather than coding and splitting this to a separate configuration file (for now).

Reply:
Done